sdk/examples/client/block/custom_inputs.rs

```diff
@@ -49,7 +49,7 @@ async fn main() -> Result<()> {
         .with_input(UtxoInput::from(output_ids_response.items[0]))?
         //.with_input_range(20..25)
         .with_output(
-            "atoi1qzt0nhsf38nh6rs4p6zs5knqp6psgha9wsv74uajqgjmwc75ugupx3y7x0r",
+            &Bech32Address::try_from_str("atoi1qzt0nhsf38nh6rs4p6zs5knqp6psgha9wsv74uajqgjmwc75ugupx3y7x0r")?,
```

Comment:
Is there really nothing we can do about that? Passing an &str is so convenient for these kind of things...

Reply:
I made some ergonomics changes. Hopefully they are agreeable.

Reply:
Thanks, much better!